### PR TITLE
Fix test countUniqueBlockAuthors

### PR DESCRIPTION
**Pull request by tmpolaczyk**
This should fix some of the spurious CI failures.

### CODE REVIEW

File: test/suites/para/test_tanssi_containers.ts [hunk 1]
```diff
@@ -394,7 +396,7 @@ async function countUniqueBlockAuthors(paraApi, blockStart, blockEnd, numAuthors
 
   let uniq = [...new Set(actualAuthors)];
 
-  if (uniq.length != numAuthors) {
+  if (uniq.length <= numAuthors) {
```
Comment (girazoki) [inline test/suites/para/test_tanssi_containers.ts:399]: where are we checking that at least we have 2 authors?

Comment (girazoki) [reply test/suites/para/test_tanssi_containers.ts:399]: I mean we should check that numAuthors > 1 as well right?

Comment (tmpolaczyk) [reply test/suites/para/test_tanssi_containers.ts:399]: We could, but in one test we check that 5 consecutive blocks have 2 authors, so there will be a small probability of failure if one of them fails to produce 2 or 3 blocks in a row.

Comment (tmpolaczyk) [reply test/suites/para/test_tanssi_containers.ts:399]: Also I put the wrong `if` condition, it should be negated

Comment (girazoki) [reply test/suites/para/test_tanssi_containers.ts:399]: I'd say for now that is a very small probability, I would add the >1 author check. If we keep getting failures, we will evaluate, but at least we check that they take somewhat turns

**Review by girazoki — APPROVED** [review]

**Pull request merged**